src/languages/de-DE.json

```
@@ -0,0 +1,278 @@
+	"slowmode_invalid_6h": "{emote_warn} Die Slowmode Geschwindigkeit kann nicht höher als **6 hours** sein!",
+
+	"slowmode_success_remove": "{emote_success} Der Slowmode wurde von folgendem Channel entfernt: {channel}",
+	"slowmode_success": "{emote_success} Der Slowmode wurde erfolgreich in {channel} zu **{slowmode}** gesetzt",
+
+	"tempban_invalid_0s": "{emote_warn} Die Länge kann nicht unter **0 sekunden** sein!",
+	"tempban_invalid_1y": "{emote_warn} Die Länge kann nicht über **einem Jahr** sein!",
+
+	"lockdown_not_boolean": "{emote_warn} Argument `lock` value must be a Boolean! (`true` or `false`)",
+	"lockdown_locked": "{emote_success} Der Kanal {channel} wurde erfolgreich gesperrt",
+	"lockdown_unlocked": "{emote_success} Erfolgreich folgenden Kanal entsperrt: {channel}",
+
+	"mute_muterole_not_found": "{emote_warn} Die Mute Rolle wurde nochnicht konfiguriert!\n{emote_wrench} **Tip:** So kannst du die Mute Rolle Konfigurieren: `!configure mute_role`",
+	"mute_success": "{emote_success} Muted **{target_tag}** `({target_id})` für **{reason}** `[#{infractionId}]`",
+	"mute_not_muted": "{emote_warn} Der User ist nochnicht gemutet!",
+	"mute_already_muted": "{emote_warn} Der User ist schon gemutet!",
+
+	"warn_success": "{emote_success} **{user_name}#{user_discriminator}** `({user_id})` wurde für den folgenden Grund verwarnt: **{reason}** `[#{infractionId}]`",
+
+	"unmute_success": "{emote_success} **{target_tag}** `({target_id})` Wurde für Folgenden Grund entmutet: **{reason}** `[#{infractionId}]`",
+
+	"infraction_not_found": "{emote_warn} Verwarnung **#{infractionId}** Konnte nicht gefunden werden!",
+	"infraction_delete_success": "{emote_success} Verwarnung **#{infractionId}** wurde erfolgreich gemutet!",
+	"infraction_info_inf_id": "**Infraction ID:** {infractionId}\n",
+	"infraction_info_target": "**User:** {target_tag} `({target_id})`\n",
+	"infraction_info_moderator": "**Moderator:** {moderator_tag} `({moderator_id})`\n",
+	"infraction_info_created": "**Erstellt:** {timestamp}\n",
+	"infraction_info_active": "**Aktiv:** {emoji}",
+	"infraction_info_expires": "**Läuft ab:** {timestamp}",
+	"infraction_info_reason": "\n\n**Grund:** {reason}",
+	"infraction_list_not_found": "{emote_warn} Der Server hat noch keine Verwarnungen!",
+	"infraction_claim_success": "{emote_success} Du bist nun für folgende Verwarnung zuständig: **#{infractionId}**",
+	"infraction_update_success": "{emote_success} Verwarnung **#{infractionId}** wurde editiert!",
+	"infraction_invalid_arguments": "{emote_warn} Fehlendes Argument `action`, Es kann eins der Folgenden sein: `delete|remove`, `info`, `list|all`, `msearch|modsearch`, `osearch|offendersearch`, `search`, `claim`, `update`",
+
+	"invalid_snowflake": "{emote_warn} `{snowflake}` is an invalid snowflake",
+	"snowflake_info": "Ich weiß nicht ob irgendjemanden oder irgenetwass diese Schneeflocke gehört\n\n",
```

"Snowflake" is an independent term with no German translation https://en.wikipedia.org/wiki/Snowflake_ID
```suggestion
	"snowflake_info": "Ich weiß nicht, ob diese Snowflake ID jemandem or etwas gehört\n\n",
```